data.sql

```diff
@@ -1,5 +1,7 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals VALUES
+(1, 'Agumon', '2020-02-03', 0, true, 10.23),
+(2, 'Gabumon', '2018-11-15', 2, true, 8),
+(3, 'Pikachu', '2021-01-07', 1, false, 15.04),
+(4, 'Devimon', '2017-05-12', 5, true, 11);
```

Comment:
- [OPTIONAL] It would be better to specify the position of each column in the table before the insert. 👍🏻  Example:

   ```sql
    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
      VALUES ('Agumon', '2020/2/3', '0', 'true', '10.23');
  
    ...
   ```